go/kbfs/libkbfs/block_ops_test.go

```diff
@@ -92,6 +93,7 @@ type testBlockOpsConfig struct {
 	initModeGetter
 	clock                       Clock
 	reporter                    Reporter
+	subsciptionManager          SubscriptionManager
```

Comment:
ouch, I guess we're committed to the spelling mistake now ;)

Reply:
Oops, just added a new commit to fix all these.